src/shapes/object.class.js

```diff
@@ -866,6 +871,7 @@
             globalCompositeOperation: this.globalCompositeOperation,
             skewX:                    toFixed(this.skewX, NUM_FRACTION_DIGITS),
             skewY:                    toFixed(this.skewY, NUM_FRACTION_DIGITS),
+            erasable:                 this.erasable
```

Comment:
by default to JSON exports only properties that influence the rendering of the objects.
Everything that is related to behaviour of objects isn't exported.
You can add this in the mixin wrapping toObject

```js
var toObject = fabric.Object.prototype.toObject;
fabric.Object.prototype.toObject = function(additionalProperties) {
  return toObject.call(this, ['erasable'].concat(additionalProperties));
}
```

Reply:
DONE